core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -823,7 +809,7 @@ object ConsumerGroupCommand extends Logging {
           case (topicPartition, newOffset) => (topicPartition, new OffsetAndMetadata(newOffset))
         }
       } else if (opts.options.has(opts.resetToDatetimeOpt)) {
-        val timestamp = convertTimestamp(opts.options.valueOf(opts.resetToDatetimeOpt))
+        val timestamp = Utils.getDateTime(opts.options.valueOf(opts.resetToDatetimeOpt))
```

Comment:
I am must wondering if we can have a dependency from `core` to `clients` module? \cc @ijuma

Reply:
@mjsax @guozhangwang can you please suggest where we can put this common code.

Reply:
`core` does depend on `clients` generally.

Reply:
Thanks for clarification. So we are good.